app/models/invoice.rb

```diff
@@ -47,6 +47,25 @@ class Invoice < ActiveRecord::Base
 
   private
 
+  def self.period(period)
+    case period
+    when 'current-month'
+      where('invoices.ends_at between ? AND ?', Date.today.beginning_of_month, Date.today.end_of_month)
```

Comment:
@yunixon it's better t use `Date.current`. 
http://makandracards.com/makandra/4601-use-time-current-date-current-datetime-current
http://stackoverflow.com/questions/6635363/what-is-the-difference-between-date-current-and-date-today

Please fix it in all places.